CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 1.2.2 (In Progress)
 ===================
 
+## Changes
+
+- Added a new API for customizing the displayed name for incoming calls in CallKit on iOS.
+  See `voice.setIncomingCallContactHandleTemplate` and the associated API docs for more information.
```

Comment:
Can we get a link to the docs?

Reply:
Since it's a new API, the autogenerated API docs do not include this API in our `latest` tag yet. When we do the release, I'll make sure to link to the API docs.